docs/UserGuide.md

```diff
@@ -72,19 +72,27 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding an employee: `addE`
 
-Adds a person to the address book.
+Adds an employee to the employees list.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `adde n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+An employee can have any number of tags (including 0)
 </div>
 
+Acceptable values for each parameter:
+* Name: John Doe / Alex / A. P. Mary
+* Phone number: +65 9123 4567 / 62353535 
+* Email: john.doe@gmail.com 
+* Address: Any string resembling a location
+* Tag (optional): Any string
+
+
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `adde n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+* `adde n/Betsy Crowe t/Junior SWE e/betsycrowe@example.com a/Brick street p/+65 9123 4567 t/Employee of the month`
```

Comment:
Are the 'e's in `adde` supposed to be capitalised?

Reply:
Yes, thanks for spotting that, it is supposed to be `addE`.